src/App.tsx

```diff
@@ -0,0 +1,72 @@
+import { useState, useEffect } from "react";
+import "./App.css";
+import { MusicLibrary } from "./components";
+import data from "./fake-data/albums.json";
+
+export interface RawData {
+  title: string;
+  artist: string;
+  artworkUrl: string;
+}
+
+export interface CleanData {
+  title: string;
+  artist: string;
+  artworkUrl: string;
+  imageAlt: string;
+}
+
+const App = () => {
+  const [albumsData, setAlbumsData] = useState<CleanData[]>([]);
+
+  const handleRemoveButtonClick = (index: number) => {
+    let newArray = [...albumsData];
+    newArray.splice(index, 1);
+    setAlbumsData(newArray);
+  };
+  const handleSortByTitleClick = () => {
+    let newData = [...albumsData];
+    newData.sort((a, b) => a.title.localeCompare(b.title));
+    setAlbumsData(newData);
+  };
+  const handleSortByArtistClick = () => {
+    let newData = [...albumsData];
+    newData.sort((a, b) => a.artist.localeCompare(b.artist));
+    console.log("howdy");
+    setAlbumsData(newData);
+  };
+
+  useEffect(() => {
+    let cleanedData: CleanData[] = [];
+    //format data to add alt field
+    data.forEach((album: RawData) => {
+      const lastSlashIndex = album.artworkUrl.lastIndexOf("/");
+      let altString =
```

Comment:
No reason this couldn't be a `const`. We always prefer a `const` to a `let` if it's possible. It let's a developer know that this value will not change in the future, while a `let` implies this will be changed (like `cleanedData`).